ci/ci.sh

```diff
@@ -148,6 +148,7 @@ case $HOST_TARGET in
     MIRI_TEST_TARGET=aarch64-linux-android  run_tests_minimal $VERY_BASIC hello panic/panic
     MIRI_TEST_TARGET=wasm32-wasi run_tests_minimal $VERY_BASIC wasm
     MIRI_TEST_TARGET=wasm32-unknown-unknown run_tests_minimal $VERY_BASIC wasm
+    MIRI_TEST_TARGET=x86_64-unknown-illumos run_tests_minimal libc-misc concurrency/simple pthread-threadname libc-getentropy libc-getrandom
```

Comment:
```suggestion
    MIRI_TEST_TARGET=x86_64-unknown-illumos run_tests_minimal $VERY_BASIC hello libc-misc concurrency/simple threadname libc-getentropy libc-getrandom
```
VERY_BASIC should always be tested, and the docs say `hello` should work so let's test that.

Also can you move this up next to Android?